invenio/templates/deployments/worker.yaml

```diff
@@ -145,7 +145,7 @@ spec:
         command: [
           "/bin/bash",
           "-c",
-          "celery -A {{ .Values.worker.app }} worker --beat -l {{ .Values.worker.log_level }}  -s {{ .Values.worker.celery_schedule }} --pidfile {{ .Values.worker.celery_pidfile }}"
+          "celery -A {{ .Values.worker.app }} beat -l {{ .Values.worker.log_level }} -s {{ .Values.worker.celery_schedule }} --pidfile {{ .Values.worker.celery_pidfile }}"
```

Comment:
I am totally ok with this, we just kept the worker in other prod instances to avoid to "waste" a POD only for the beat and also have an extra worker, but making sure that the `replica` count will be always only `1`.